nautobot/dcim/api/views.py

```diff
@@ -748,7 +749,7 @@ class VirtualChassisViewSet(ModelViewSet):
 #
 
 
-class PowerPanelViewSet(ModelViewSet):
+class PowerPanelViewSet(ModelViewSet, NotesViewSetMixin):
```

Comment:
PowerPanel is a PrimaryModel - this should *probably* become a NautobotModelViewSet.

Reply:
Thanks! I was trying to find these as I made the changes. I missed this one.